### PR TITLE
21.1.5 update

### PR DESCRIPTION
**Pull request by friedcucumber**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,24 +1,35 @@
...
-<br/>
-
-
+<!-- default badges list -->
+[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T326792)
+[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
+<!-- default badges end -->
+# WPF Dock Layout Manager - Populate Tabs with Items
+
+This example displays content in [DockLayoutManager](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.DockLayoutManager)'s tabs in different approaches.
```
Comment (albertov05) [inline Readme.md]: Please update this sentence without using "approaches".

File: Readme.md [hunk 2]
```diff
@@ -1,24 +1,35 @@
...
+
+<img src="https://user-images.githubusercontent.com/12169834/175351620-63365ade-c0a0-4bd0-ac6c-0907dd5a3647.png" width=525px/>
+
+To do this, you can use any of the following containers:
+
+- A [LayoutGroup](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.LayoutGroup) with [GroupBorderStyle](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.LayoutGroup.GroupBorderStyle) set to `Tabbed`
+- A [TabbedGroup](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.TabbedGroup)
+- A [DocumentGroup](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.DocumentGroup)
+
+A [DocumentGroup](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.DocumentGroup) allows you to display dynamic content. It works similar to the document area in the Visual Studio.
```
Comment (albertov05) [inline Readme.md]:
```suggestion
A [DocumentGroup](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.DocumentGroup) allows you to display dynamic content. It works similarly to the document area in Visual Studio.
```

File: Readme.md [hunk 3]
```diff
@@ -1,24 +1,35 @@
...
+
+![](https://raw.githubusercontent.com/DevExpress-Examples/how-to-display-items-in-tabs-t326792/15.1.3+/media/cdfa2c50-a7ba-11e5-80bf-00155d62480c.png)
+
+A [TabbedGroup](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.TabbedGroup) displays static content like toolbars, service information, an other.
```
Comment (albertov05) [inline Readme.md]:
```suggestion
A [TabbedGroup](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.TabbedGroup) displays static content such as toolbars, service information, and so on.
```

File: Readme.md (hunk 1 above)
Comment (friedcucumber) [reply Readme.md]:
```suggestion
This example displays content in [DockLayoutManager](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.DockLayoutManager)'s tabs.

```

**Pull request merged**